sdk/qnamaker/Azure.AI.QnaMaker/src/autorest.md

```diff
@@ -0,0 +1,46 @@
+# Azure.AI.QnaMaker
+
+Run `dotnet build /t:GenerateCode` to generate code.
+
+## AutoRest Configuration
+
+We refer directly to the OpenAPI document to avoid batch processing of multiple tags (commented lines not working).
+
+See <https://aka.ms/autorest> for more information.
+
+``` yaml
+# require:
+# - https://github.com/Azure/azure-rest-api-specs/blob/0b9087be79feaca4504b1ecb277875bc6be56617/specification/cognitiveservices/data-plane/QnAMaker/readme.md
+# batch: false
+# tag: release_5_0_preview.2
```

Comment:
@ShivangiReja @pakrym this and variants didn't work: all tags kept getting processed. I also tried overriding batch has a hash of just 1 tag but that didn't work either, so I ended up referring directly to the JSON file.

@johanste it's worth pointing out they have C# and other language configuration in their _readme.md_ that is track 1 and stuff we probably don't want, if even applicable to some or all generators. Makes me wonder if we want to introduce a new, fresh file and start over with client generation configuration. We could even just add a new tag. Either way, I think we still use the same swaggers - though we need to parameterize the api-version that is part of the path.